### PR TITLE
Adding documentation for learning rate schedule

### PR DESCRIPTION
**Pull request by anupambhatnagar**
### Proposed change(s)

Added a description of `learning_rate_schedule` in the docs 

### Useful links (Github issues, JIRA tickets, ML-Agents forum threads etc.)

https://github.com/Unity-Technologies/ml-agents/issues/3328
https://jira.unity3d.com/browse/MLA-751

### Types of change(s)

- [ ] Bug fix
- [ ] New feature
- [ ] Code refactor
- [ ] Breaking change
- [x] Documentation update
- [ ] Other (please describe)

### Checklist
- [ ] Added tests that prove my fix is effective or that my feature works
- [ ] Updated the [changelog](https://github.com/Unity-Technologies/ml-agents/blob/master/com.unity.ml-agents/CHANGELOG.md) (if applicable)
- [ ] Updated the [documentation](https://github.com/Unity-Technologies/ml-agents/tree/master/docs) (if applicable)
- [ ] Updated the [migration guide](https://github.com/Unity-Technologies/ml-agents/blob/master/docs/Migrating.md) (if applicable)

### CODE REVIEW

File: docs/Training-ML-Agents.md [hunk 1]
```diff
@@ -190,6 +190,7 @@ example environments are included in the provided config file.
 | init_entcoef         | How much the agent should explore in the beginning of training.                                                                                                                         | SAC                      |
 | lambd                | The regularization parameter.                                                                                                                                                           | PPO                      |
 | learning_rate        | The initial learning rate for gradient descent.                                                                                                                                         | PPO, SAC             |
+| learning_rate_schedule | Determines how learning rate changes over time. Options: linear (default), constant | PPO, SAC |
```
Comment (ervteng) [inline docs/Training-ML-Agents.md]: The defaults are actually different for PPO and SAC - would just leave out the options as they're specified within the PPO/SAC docs

Comment (anupambhatnagar) [reply docs/Training-ML-Agents.md]: should we provide a link to the PPO/SAC docs here?

Comment (ervteng) [reply docs/Training-ML-Agents.md]: I think there's a link at the top of the table

Comment (anupambhatnagar) [reply docs/Training-ML-Agents.md]: changes made. can you please approve?
